README.md

```diff
@@ -28,6 +28,15 @@ Alternatively,
 1.  Clone the `khronos/spirv-3.6.1` branch.
 2.  Follow [LLVM build instructions](http://llvm.org/docs/GettingStarted.html).
 
+Test instructions
+-----------------
+
+All tests related to the translator are placed in [test/SPIRV](https://github.com/KhronosGroup/SPIRV-LLVM/tree/khronos/spirv-3.6.1/test/SPIRV).
+Execute the following command to run translator tests:
+```
+make check-llvm-spirv
```

Comment:
I don't think this works.

Probably should be

llvm-lit -v -v ${LLVM_SRC_DIR}/test/SPIRV

where ${LLVM_SRC_DIR} is the LLVM source directory.

Reply:
It works. You want I change it to method you provided?

Reply:
I just tried it with my build

$ make check-llvm-spirv
make: *** No rule to make target 'check-llvm-spirv'.  Stop.

I guess it may requires clang to work, whereas my build does not include clang.

llvm-lit always works.

Reply:
Sorry, you are right. It works with newer LLVM only. Updated patch.